src/main/java/com/minecolonies/api/util/FoodUtils.java

```diff
@@ -17,7 +18,12 @@ public class FoodUtils
      */
     public static boolean canEat(final ItemStack stack, final int buildingLevel)
     {
-        return buildingLevel < 3 || stack.getItem().getFoodProperties(stack, null).getNutrition() >= buildingLevel + 1;
+        if (buildingLevel < 3)
+        {
+            return stack.getItem().getFoodProperties(stack, null) != null;
+        }
+        final FoodProperties foodProperties = stack.getItem().getFoodProperties(stack, null);
+        return foodProperties != null && stack.getItem().getFoodProperties(stack, null).getNutrition() >= buildingLevel + 1;
```

Comment:
Should use `foodProperties` rather than fetching again.

Reply:
yes